tests/suites/0_stateless/00_dummy/00_0000_dummy_select_1.sql

```diff
@@ -16,3 +16,6 @@ SELECT b.number FROM (SELECT * FROM numbers(3) AS b) AS a ORDER BY b.number; --
 SELECT number, 'number', "number" FROM numbers(3) AS a order by a.number;
 SELECT 'That\'s good.';
 SELECT *; -- {ErrorCode 1015}
+SELECT DISTINCT * FROM numbers(3) order by number;
```

Comment:
It would be better move the test to `suites/0_stateless/03_dml` like a new name: `03_0021_select_distinct.sql`.